CoreFoundation/Base.subproj/CFBase.h

```diff
@@ -70,6 +70,10 @@
 
 #if __BLOCKS__ && ((TARGET_OS_MAC && !(TARGET_OS_EMBEDDED || TARGET_OS_IPHONE)) || (TARGET_OS_EMBEDDED || TARGET_OS_IPHONE))
 #include <Block.h>
+#endif
+
+#if defined(__FreeBSD__)
```

Comment:
This should really be DEPLOYMENT_TARGET_FREEBSD (and then that should be defined in terms of __FreeBSD__.)